Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.13.9-alpine3.11 as builder
+
+WORKDIR /go/github.com/doitintl/kube-no-trouble
```

Comment:
can we move this to smth like just `/src`? This is not a "correct" path with old `$GOPATH` scheme anyway (missing `src/` bit), and go historically has split-brain behavior with $GOPATH and modules, I would prefer to make it clear

Reply:
sure :)